### ISSUE TITLE
Discord Bot with Q's to direct new members

### ISSUE THREAD

**Issue by danelsuga** [CommonsBuild/coordination#884]
## 👇🏽  What is this issue about? Give as much detail as you can. 
Put in place a Discord bot to help collect info form new members and direct them appropriately.

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)
1. New members will be organized upon arrival and will be able to find their way to best places for contributing and getting settled in the community.
2.
3.

## 🧍🏻‍♀️🧍🏻‍♂️  Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)
@eduardovegap 
@Vyvy-vi 
@MrsBadgerface 

## 👀  Who should be in the loop about this issue?
@

**Comment by danelsuga** [comment]
@Vyvy-vi has this ready to go, and we should be implementing it very soon.

**Comment by Vyvy-vi** [comment]
The only thing that's left to discuss is where do we want this data to go

**Comment by tamarandom** [comment]
Good question! What would you propose? Perhaps a Google Sheet would suffice?

**Comment by danelsuga** [comment]
@tamarandom @Vyvy-vi 

We’re choosing a private Discord channel in the end, I think, as it is easiest.

**Comment by Vyvy-vi** [comment]
This has been implemented :)

**Comment by tamarandom** [comment]
Is the data collected in a form that can be searched or otherwise parsed. A few cases come to mind on how that would be very valuable for the TEC in the future.

**Comment by danelsuga** [comment]
@Vyvy-vi 
@tamarandom 
@liviade1 
@natenatesoo 
We should do advice process on this since it impacts things that are unseen. 

For the future TEC, we probably need a way to parse and understand this data so we can use it and, for example, send contributors down different channels. 

In Discord, the data appears to be just in a list. Maybe a spreadsheet would be better? And the questions designed in a way that allows for keywords to be used that can then be markers for distributing the data as needed. 

What do you's think?

**Comment by danelsuga** [comment]
Here is Vy's video showing how it works: 

https://file.coffee/u/J5ZdAky_0djEKL.mp4

And here is the Discord channel where the data is dropped: 

#orientation-data

**Comment by Vyvy-vi** [comment]
If we want a spreadsheet, I could make the data go to Airtable. :+1:
For categorising, could we take some of the major keywords, and have a question that is a Choice Question with options?

**Comment by danelsuga** [comment]
Those sound like great ideas, @Vyvy-vi. Looking forward to feedback from others. @liviade1 @tamarandom

**Comment by tamarandom** [comment]
Thanks @Vyvy-vi and @danelsuga! If you recommend Airtable, @Vyvy-vi, then let's go with it. I am not very  familiar with it and think whatever is the simplest solution to create a searchable archive should work. 

For the user journey - are they able to "opt-out" of this questionnaire? So if they decide not to answer these questions, will they still see the server channels?

**Comment by eduardovegap** [comment]
@tamarandom yes they can! This is a bot that gets send as a personal message, so it doesn't affect the user journey.

**Comment by danelsuga** [comment]
This is from @saunic :

👉  **Add more questions to orientation data collection bot:**
🔹  What are your interests?
🔹 What are skills you have that you enjoy using?
🔹 What interested you in joining?
🔹  Which timezone are you in?
🔹  How much time are you interested in contributing, if you want to contribute?
*This way we know exactly how to direct EVERY new user right away, specific to their needs vs generalised answers.*

👉 **Prepare slide deck for orientation call. Give complete information for those who want to contribute right away:**
🔹 What is the trusted seed? Join the trusted seed.
🔹 Working group summaries including:
- description of focus
-wishlist
-how can I be a part of that working group? include info such as:
--- article/video tutorial for specific tools used by this wg
--- when are meetings usually held?
--- who is the steward/ person to reach out to?
--- tasks categorised by time availability (0-5hrs, 6-9 hrs, 10-14 hrs, 15+ hours)
🔹 Link to gitbook
🔹Discord tutorial
🔹 Zenhub tutorial
🔹 Link to zenhub
🔹 When is the next community call? Where can the recording be found if the time doesn't work for them?
*This way they won't have to click multiple links in the middle of the orientation call. All information needed is presented on the screen, maintaining focus and ease for users new to discord. 
They will also know what to expect to know by the time the orientation call is finished and question time can be for more in depth questions/questions not answered by slide deck.*

👉 **Make bot tell users to join orientation call (including when and where) AND share the slide deck beforehand**
*This way, if they miss the call or can't make it, they have the slide deck as a source of info and can get started on taking action/contributing right away.*

**Comment by danelsuga** [comment]
Advice Process requested from @liviade1 @solsista

**Comment by MrsBadgerface** [comment]
Back from holiday and catching up! Thanks so much @Vyvy-vi! This is brilliant. I love how the options open up. Is there a way we can see all the pathways? depending on what choices people make? (i.e. if there're here for learning, or for community etc.)?

We have a Communitas WG miro board section here: https://app.zenhub.com/workspaces/tec-coordination-workspace-5fad0d3fbbe4da0011c2f40d/board?repos=306706322,349409011

Perhaps that would be a good place to map out all the different choices and what new information comes up per choice, with screen shots showing each of the new text boxes or video clips that open up, so that we can adjust if needed and see what's missing?

I also think we need another place for us to see the information gathered on new members will be key to making that information usable for WG members, whether airtable or google spreadsheet. Actually, @danelsuga @eduardovegap (I'd add Nic but don't know her handle here?) what if we used an option that allowed multiple sheets that's easy to navigate (I'm guessing that might be Google sheets)? We could then create a multi-sheeted WG database, which along with the Miro board would act as this WG's shared documents. Maybe we could have one sheet for WG tasks and roles (what's available, which roles within the WG itself are suitable for contributors - split into time commitment levels - and which need a bit of history and experience within the TEC first), one sheet for new joiner information gathered via these bots, one sheet for budget planning etc.

If we do gather information in a second place, we'd need to add a sentence somewhere along the lines of 'All information gathered will be accessible to all TEC members. Contact a steward to get your information removed at any time' before we gather it.

I love Nic's suggestions! Definitely will need a way to map out and organise these different pathways depending on what information they're giving us via the collection bot...

**Comment by danelsuga** [comment]
Here's nic! @saunic

**Comment by MrsBadgerface** [comment]
Thanks @danelsuga 

Also, having now read @saunic's dashboard idea, I think my comments are all moot points! Sounds like the dashboard would automate all of this info :)

**Comment by tamarandom** [comment]
@Vyvy-vi is, I believe, working on the backend of this so assigning it to him.

**Comment by tamarandom** [comment]
This is being rolled into the new discord onboarding journey!
